openapi-v1.yaml

```diff
@@ -6299,40 +6231,7 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-  /groups/{namespace}:
-    parameters:
-      - name: namespace
-        type: string
-        in: path
-        required: true
-        description: The namespace to operate on
-    get:
-      description: Returns one page of top level groups in namespace.
-      operationId: listTopLevelGroups
-      tags:
-        - groups
-      parameters:
-        - name: page
-          in: query
-          description: pagination offset
-          type: integer
-          required: false
-        - name: per_page
-          in: query
-          description: pagination limit
-          type: integer
-          required: false
-      responses:
-        200:
-          description: the group contents
-          schema:
-            $ref: '#/definitions/GroupListing'
-        default:
-          description: error response
-          schema:
-            $ref: "#/definitions/Error"
-
-  /groups/{namespace}/{name}:
+  /groups/{namespace}/{name}/contents:
```

Comment:
Style: all these search parameters will be used only by GET no matter how the API evolves. I suggest to keep the path parameters here and move the query params under `get:`

Reply:
Thanks, also eliminated duplicates `page`, `per_page`